download.html

```diff
@@ -98,8 +98,8 @@ <h3>Latest stable release: {{versionFull}} ({{date}})</h3>
 		Other packages and versions are available <a href="#versionList">below</a>. For an overview of the current maintenance state of the different OpenUI5 versions, please check the <a href="https://openui5.hana.ondemand.com/versionoverview.html">version overview page</a>.
 
 		<div id="noUnstableVersion" style="display: none">
```

Comment:
i think you have to remove the "display:none" and the logic in the script that evaluates the beta flag as well

Reply:
done